rpm/qm.spec

```diff
@@ -223,6 +223,9 @@ install -d %{buildroot}%{_sysconfdir}/qm/containers/containers.conf.d
 # START - qm dropin sub-package - mount kvm            #
 ########################################################
 %if %{enable_qm_mount_bind_kvm}
+    # first step - add drop-in file in /etc/containers/containers.d.conf/qm_dropin_mount_bind_snd.conf
```

Comment:
If gate is passing, it means that this part is failing the CI
I do not think it is important
That one below is enough 
```
%{_builddir}/qm-%{version}/etc/qm/containers/containers.conf.d/qm_dropin_mount_bind_kvm.conf %{buildroot}%
```

Reply:
@Yarboa If you say, I trust. I was just cascading /dev/kvm from host -> QM -> nested containers.

Reply:
It has improved the status of that #660 
I am testing manually but i have selinux issues due to disk partition extension